ElectronicsCrashCourse.md

```diff
@@ -0,0 +1,241 @@
+
+### Electrical system diagram
+
+![Rev electrical system diagram](<Assets/RevElectricalSystemDiagram.png>)
+
+This diagram shows many of the electrical components found on a typical FRC robot.
+You don't need to memorize this layout, but its handy to have this as a reference.
+
+### Battery
+
+![Battery](Assets/Battery.jpg)
+
+All of the power our robot uses comes from a single 12-volt car battery.
+These can be found on our battery cart in the corner of the lab.
+Not all batteries are created equal: Some brands and specific batteries can hold more juice for longer than others.
+One way we can see the performance of a battery is the resting voltage (or voltage when no motors or similarly power hungry devices are running) of the battery after it has charged.
+This can be checked with the battery beak or on the driver station.
+A good battery will have a resting voltage of over 13 volts, and any battery we use will be above 12.
+As the battery is used the voltage will go down.
+We tend to swap our batteries after they get to around 11 and a half volts, although if we are testing something sensitive to voltage (like auto) we might swap them sooner.
+After a match a good battery is usually still above 12 volts.
+To track batteries so we know which batteries are "good" and which are "bad" we give them names like Garbanzo or W.I.S. (Women In STEM).
+
+When the robot is on the voltage will also drop as power is actively used by mechanisms.
+This is called voltage sag.
+This means that we can't pull full power from every motor on the robot at once and have to be careful to design and program around our power budget.
+When the voltage of the robot drops too low we can brownout.
+This is usually first visible when the LEDs on the robot go out.
+When the voltage is low enough motors will begin to be turned off which is visible as a stuttering or jerking.
+If the voltage is even lower, the robot can restart.
+Needless to say we want to avoid any brownout conditions in a match or when testing.
+
+### Main Breaker
+
+![Main Breaker](Assets/Breaker.jpg)
+
+The main breaker is the power switch of the robot.
+The red button will turn the robot off, and a small black lever on the side of the breaker will turn it on.
+The main breaker is directly connected to the battery and limits the current draw of the robot, or the amount of power it can use at once.
+Whenever you need to be hands on with the robot, make sure the breaker is off.
+If you can't connect to the robot, make sure it's on.
+
+The breaker should be mounted in a visible and accessible location on all robots, although it tends to blend in with its black casing.
+We tend to have a 3dprinted guard over the off switch to prevent accidental presses by other robots mid-match.
+
+### Power Distribution Hub (PDH)
+
+![PDH](Assets/PDH.webp)
+
+The PDH takes the power from the battery and distributes it to the motors, sensors, and other components on the robot (Almost like its a hub for distributing power!).
```

Comment:
You might want to mention that the PDP exists too as it might show up in other documentation they read.